SPECS/bcc+clang.spec

```diff
@@ -14,7 +14,7 @@ Source1:        http://llvm.org/releases/%{llvmver}/llvm-%{llvmver}.src.tar.xz
 Source2:        http://llvm.org/releases/%{llvmver}/cfe-%{llvmver}.src.tar.xz
 
 BuildArch:      x86_64
-BuildRequires:  bison, cmake >= 2.8.7, flex, gcc, gcc-c++, libxml2-devel, python2-devel
+BuildRequires:  bison, cmake >= 2.8.7, flex, gcc, gcc-c++, libxml2-devel, python2-devel, libelf-dev
```

Comment:
Also, looking at the details of elfutils-libelf-devel, this is going to be an extra runtime dependency as well. Might need to switch it to elfutils-libelf-devel-static.